fastly/block_fastly_service_director.go

```diff
@@ -301,3 +318,31 @@ func flattenDirectors(directorList []*gofastly.Director, directorBackendList []*
 	}
 	return dl
 }
+
+func getDirectorBackendChange(d *schema.ResourceData, resource map[string]interface{}) (odb *schema.Set, ndb *schema.Set) {
+	name := resource["name"]
```

Comment:
I'd prefer to keep the variable initialisation near to where it is used (see the following suggested change).

```suggestion
```